### PR TITLE
HTML img: Created first draft

### PR DESCRIPTION
**Pull request by nebrelbug**
Basic introduction to img tags

### CODE REVIEW

**Comment by nebrelbug** [comment]
Oops, I put in that img tags close with ```<``` instead of ```>```.

File: src/pages/html/images/index.md [hunk 1]
```diff
@@ -8,6 +8,28 @@ This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/
 <a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
 
 <!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+A simple HTML Image element can be included in an HTML document like this:
+
+```
```
Comment (texas2010) [inline src/pages/html/images/index.md]: fix this. ```html

File: src/pages/html/images/index.md [hunk 2]
```diff
@@ -8,6 +8,28 @@ This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/
...
+<img src="path/to/image/file" alt="this is a cool picture>
+```
+
+```alt``` tags provide alternate text for an image. One use of the ```alt``` tag is for visually impaired people using a screen reader; they can be read the ```alt``` tag of the image in order to understand the image's meaning.
```
Comment (texas2010) [inline src/pages/html/images/index.md]: `alt`

File: src/pages/html/images/index.md [hunk 3]
```diff
@@ -8,6 +8,28 @@ This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/
...
+
+Note that the path to the image file can be either relative or absolute. Also, remember that the ```img``` element is self-closing, meaning that it does not need a closing ```</img>``` tag and instead closes with just a single ```<```.
+
+Example:
+
+```
```
Comment (texas2010) [inline src/pages/html/images/index.md]: use ```html

File: src/pages/html/images/index.md [hunk 4]
```diff
@@ -8,6 +8,28 @@ This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/
...
+<img src="https://example.com/image.png">
+```
+
+is the same as:
+
+```
```
Comment (texas2010) [inline src/pages/html/images/index.md]: use ```html

File: src/pages/html/images/index.md [hunk 5]
```diff
@@ -8,6 +8,28 @@ This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/
...
 <!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+A simple HTML Image element can be included in an HTML document like this:
+
+```
+<img src="path/to/image/file" alt="this is a cool picture>
+```
+
+```alt``` tags provide alternate text for an image. One use of the ```alt``` tag is for visually impaired people using a screen reader; they can be read the ```alt``` tag of the image in order to understand the image's meaning.
+
+Note that the path to the image file can be either relative or absolute. Also, remember that the ```img``` element is self-closing, meaning that it does not need a closing ```</img>``` tag and instead closes with just a single ```<```.
```
Comment (texas2010) [inline src/pages/html/images/index.md]: \`img`

this supposes to be \``.

\`<`

Comment (nebrelbug) [reply src/pages/html/images/index.md]: Thanks! Should I just edit my pull request...?

File: src/pages/html/images/index.md [hunk 6]
```diff
@@ -8,6 +8,28 @@ This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/
 <a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
 
 <!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+A simple HTML Image element can be included in an HTML document like this:
+
+```
+<img src="path/to/image/file" alt="this is a cool picture>
```
Comment (texas2010) [inline src/pages/html/images/index.md]: you forget to add another double quote in the end and before >

File: src/pages/html/images/index.md (hunk 1 above)
Comment (nebrelbug) [reply src/pages/html/images/index.md]: I just updated, is it better now?

**Comment by nebrelbug** [comment]
I fixed all of the requested changes... I think

File: src/pages/html/images/index.md (hunk 1 above)
Comment (texas2010) [reply src/pages/html/images/index.md]: you forget to do this too.

File: src/pages/html/images/index.md [hunk 7]
```diff
@@ -8,8 +8,32 @@ This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/
...
 <!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+A simple HTML Image element can be included in an HTML document like this:
+
+```
+<img src="path/to/image/file" alt="this is a cool picture">
+```
+
+`alt` tags provide alternate text for an image. One use of the `alt` tag is for visually impaired people using a screen reader; they can be read the `alt` tag of the image in order to understand the image's meaning.
+
+Note that the path to the image file can be either relative or absolute. Also, remember that the `img` element is self-closing, meaning that it does not close with the `</img>` tag and instead closes with just a single `>`.
```
Comment (texas2010) [inline src/pages/html/images/index.md]: please \`<\/img>\` to \'\`

Comment (nebrelbug) [reply src/pages/html/images/index.md]: But I'm saying that it doesn't close with </img> like divs close with </div>, but instead closes with '>'

Comment (nebrelbug) [reply src/pages/html/images/index.md]: Thank you so much for your help! This is my first open-source contribution.

Comment (texas2010) [reply src/pages/html/images/index.md]: no problem. welcome anytime

File: src/pages/html/images/index.md (hunk 1 above)
Comment (nebrelbug) [reply src/pages/html/images/index.md]: I just fixed it to ``

**Comment by nebrelbug** [comment]
I fixed the `</img>` to `<img /`

**Comment by texas2010** [comment]
@nebrelbug thanks!

**Pull request merged**